Documentation/ApiOverview/Events/Hooks/Index.rst

```diff
@@ -6,9 +6,14 @@
 Hooks
 =====
 
-Hooks are basically places in the source code where a user function will be called for processing
-if such has been configured. While there are conventions and best practises of how hooks should be
-implemented the Hook Concept itself doesn't prevent it from being used in any way.
+Hooks are basically places in the source code where a user function will be
+called for processing, if such has been configured. While there are conventions
+and best practises of how hooks should be implemented the hook concept itself
+does not prevent it from being used in any way.
+
+Hooks are being phased-out in the TYPO3 Core and extension authors are also
```

Comment:
Hmm, that does not really help as most hooks don't have a corresponding event (apart from the deprecated ones from v12.1-v12.4).

Reply:
As far as I understand almost all Hooks outside of the datahandler have become deprecated or removed in the Core. And what is wrong about advising extension authors to introduce events and not news Hooks?

Reply:
I tried to explain it more clear